src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs

```diff
@@ -182,92 +182,80 @@ private static void ValidateRegistrations(IEnumerable<HealthCheckRegistration> r
 
         internal static class EventIds
         {
-            public static readonly EventId HealthCheckProcessingBegin = new EventId(100, "HealthCheckProcessingBegin");
-            public static readonly EventId HealthCheckProcessingEnd = new EventId(101, "HealthCheckProcessingEnd");
-
-            public static readonly EventId HealthCheckBegin = new EventId(102, "HealthCheckBegin");
-            public static readonly EventId HealthCheckEnd = new EventId(103, "HealthCheckEnd");
-            public static readonly EventId HealthCheckError = new EventId(104, "HealthCheckError");
-            public static readonly EventId HealthCheckData = new EventId(105, "HealthCheckData");
+            public const int HealthCheckProcessingBeginId = 100;
+            public const int HealthCheckProcessingEndId = 101;
+            public const int HealthCheckBeginId = 102;
+            public const int HealthCheckEndId = 103;
+            public const int HealthCheckErrorId = 104;
+            public const int HealthCheckDataId = 105;
+
+            // Hard code the event names to avoid breaking changes. Even if the methods are renamed, these hard-coded names shouldn't change.
+            public const string HealthCheckProcessingBeginName = "HealthCheckProcessingBegin";
+            public const string HealthCheckProcessingEndName = "HealthCheckProcessingEnd";
+            public const string HealthCheckBeginName = "HealthCheckBegin";
+            public const string HealthCheckEndName = "HealthCheckEnd";
+            public const string HealthCheckErrorName = "HealthCheckError";
+            public const string HealthCheckDataName = "HealthCheckData";
+
+            public static readonly EventId HealthCheckProcessingBegin = new EventId(HealthCheckProcessingBeginId, HealthCheckProcessingBeginName);
```

Comment:
Are these still used?

Reply:
Looks like all but 1 were only used by unit tests. I've moved them to the unit tests.